mapie/tests/test_classification.py

```diff
@@ -1220,7 +1220,7 @@ def test_get_last_included_proba_shape(k_lambda, strategy):
 
     mapie = MapieClassifier(estimator=clf, **STRATEGIES[strategy][0])
     include_last_label = STRATEGIES[strategy][1]["include_last_label"]
-
+    mapie.n_classes_ = len(np.unique(y))
```

Comment:
Why do you add this here?

Reply:
because as we only use the _get_last_included proba method, the n_classes_ attribute is not defined in MAPIE

Reply:
just checked, unnecessary indeed